modules/admin_manual/pages/installation/manual_installation.adoc

```diff
@@ -379,7 +370,10 @@ long as it sends the correct `X-Forwarded-Host` header.
 
 NOTE: For further information on improving the quality of your ownCloud installation, please see xref:installation/configuration_notes_and_tips.adoc[the configuration notes and tips guide].
 
-NOTE: Admins of SELinux-enabled distributions such as _CentOS_, _Fedora_, and _Red Hat Enterprise Linux_ may need to set new rules to enable installing ownCloud. See xref:installation/configuration_notes_and_tips.adoc#selinux[SELinux] for a suggested configuration.
+NOTE: Admins of SELinux-enabled distributions such as _CentOS_, _Fedora_, and
```

Comment:
Is there a specific need to split sentences over multiple lines?

Reply:
see above